generators/app/templates/__electron.electron.main.js

```diff
@@ -0,0 +1,60 @@
+const electron = require('electron')
```

Comment:
Could it be possible to use TS also here? (for a later version, JS is fine for a first release!)

Seems that can be done, as seen with https://github.com/maximegris/angular-electron, but it does not seems pretty :/